Wikipedia/Code/ProfileView.swift

```diff
@@ -0,0 +1,233 @@
+import SwiftUI
+import WMFComponents
+
+struct ProfileView: View {
```

Comment:
@l-olson1214 It looks like there's some duplicate classes here, probably as a result of us moving code files around earlier this week. I would like to remove the ProfileView.swift that lives here in the Wikipedia/Code/ directory, and keep the one in WMFComponents/Components/Profile directory. Same goes for ProfileViewController. Can you delete this file (you may need to copy it's logic over into the WMFComponents class).

Reply:
Good catch!